plugin/output/splunk/splunk.go

```diff
@@ -146,13 +146,16 @@ func (p *Plugin) out(workerData *pipeline.WorkerData, batch *pipeline.Batch) {
 
 	root := insaneJSON.Spawn()
 	outBuf := data.outBuf[:0]
+
 	for _, event := range batch.Events {
 		root.AddField("event").MutateToNode(event.Root.Node)
 		outBuf = root.Encode(outBuf)
 		_ = root.DecodeString("{}")
 	}
 	insaneJSON.Release(root)
 	data.outBuf = outBuf
+	
+	p.logger.Debugf("Trying to send: %s", outBuf)
 
 	for {
 		err := p.send(outBuf)
```

Comment:
Probably it make sense to add `p.logger.Debugf("Failed to send: %s", outBuf)` if `if err != nil`

Reply:
It is already there

Reply:
https://github.com/ozontech/file.d/pull/81/files#diff-b7d3945688c3a827787b7eeef8303aa7e0f7e5fe26fed4108068ad34f77e3301R164

Reply:
Oh yes, the debug message, I will add it, thanks

Reply:
Done